contact.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
+  <link rel="stylesheet" href="portfolio.html">
+</head>
+
+<body>
+
+  <header>
+    <h1><a href="index.html">Pilar TORRES</a></h1>
+    
+    <nav class="navegation-bar">
+      <ul class="navegation-list">
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+
+    <h3>Contact me</h3>
+
+    <div class="container">
+      <form>
+        <label for="fname">First Name</label>
+        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+        <label for="lname">Last Name</label>
+        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+        <label for="country">Country</label>
+        <select id="country" name="country">
+            <option value="australia">Spain</option>
+            <option value="canada">Germany</option>
+            <option value="usa">London</option>
+        </select>
+
+        <label for="subject">Subject</label>
+        <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+
+        <input type="submit" value="Submit">
+      </form>
```

Comment:
Well done! :+1: This looks like a well structured form! Did you take it from somewhere else or write it yourself?

Reply:
I saw something like this in the orientation curs, but also I search it yesterday to remember how can I create a contact-box.